ssh-key/tests/public_key.rs

```diff
@@ -1,6 +1,8 @@
 //! SSH public key tests.
 
 use hex_literal::hex;
+use sec1::consts::U32;
+use ssh_key::public::{Ed25519PublicKey, SkEcdsaSha2NistP256, SkEd25519};
```

Comment:
```suggestion
#[cfg(feature = "ecdsa")]
use {
    sec1::consts::U32,
    ssh_key::public::{Ed25519PublicKey, SkEcdsaSha2NistP256, SkEd25519}
};
```

Note there's an existing `#[cfg(feature = "ecdsa")]` immediately below this you could group these imports with as well.

Reply:
Fixed, thx for pointing me the right way ;).